packages/block-editor/src/components/inserter/hooks/use-block-types-state.js

```diff
@@ -26,7 +27,7 @@ import { withRootClientIdOptionKey } from '../../../store/utils';
  */
 const useBlockTypesState = ( rootClientId, onInsert, isQuick ) => {
 	const options = useMemo(
-		() => ( { [ withRootClientIdOptionKey ]: ! isQuick } ),
+		() => ( { [ noFilter ]: ! isQuick } ),
```

Comment:
While reading over this PR, it took me a little while to understand what `noFilter` meant, possibly because of the double negatives? Totally optional, but what about a positive `isFiltered`?